platform/ios/app/MBXEmbededMapViewController.m

```diff
@@ -0,0 +1,153 @@
+        [view addConstraint:[NSLayoutConstraint constraintWithItem:label attribute:NSLayoutAttributeWidth relatedBy:NSLayoutRelationEqual toItem:view attribute:NSLayoutAttributeWidth multiplier:1.0 constant:0.0]];
+        [view addConstraint:[NSLayoutConstraint constraintWithItem:label attribute:NSLayoutAttributeHeight relatedBy:NSLayoutRelationEqual toItem:view attribute:NSLayoutAttributeHeight multiplier:0.5 constant:0.0]];
+        [view addConstraint:[NSLayoutConstraint constraintWithItem:switchControl attribute:NSLayoutAttributeTop relatedBy:NSLayoutRelationEqual toItem:label attribute:NSLayoutAttributeBottom multiplier:1.0 constant:0.0]];
+        [view addConstraint:[NSLayoutConstraint constraintWithItem:switchControl attribute:NSLayoutAttributeCenterX relatedBy:NSLayoutRelationEqual toItem:view attribute:NSLayoutAttributeCenterX multiplier:1.0 constant:0.0]];
+        
+    }
+    
+    UIStackView *stackView = [[UIStackView alloc] initWithArrangedSubviews:items];
+    stackView.alignment = UIStackViewAlignmentFill;
+    stackView.distribution = UIStackViewDistributionFillEqually;
+    stackView.frame = CGRectMake(0, CGRectGetMaxY(self.navigationController.navigationBar.frame), CGRectGetWidth(self.view.frame), 65);
+    
+    [self.view addSubview:stackView];
+}
+
+- (void)didSwitch:(UISwitch *)sw {
+    [self switchContorl:sw.tag];
+    if (sw.tag == MBXEmbeddedControlRotation) {
+        
+    }
+}
+
+- (void)rotation:(UIRotationGestureRecognizer *)rotationGesture
+{
+    self.mapView.transform = CGAffineTransformRotate(rotationGesture.view.transform, rotationGesture.rotation);
+}
+
+- (void)switchContorl:(MBXEmbeddedControl) control {
+    switch (control) {
+        case MBXEmbeddedControlZoom:
+            self.mapView.zoomEnabled = !self.mapView.zoomEnabled;
+            break;
+        case MBXEmbeddedControlScroll:
+            self.mapView.scrollEnabled = !self.mapView.scrollEnabled;
+            break;
+        case MBXEmbeddedControlRotation:
+            self.mapView.rotateEnabled = !self.mapView.rotateEnabled;
+            break;
+        case MBXEmbeddedControlPitch:
+            self.mapView.pitchEnabled = !self.mapView.pitchEnabled;
+            break;
+    }
+}
+
+- (BOOL)statusForControl:(MBXEmbeddedControl) control {
+    switch (control) {
+        case MBXEmbeddedControlZoom:
+            return self.mapView.zoomEnabled;
+        case MBXEmbeddedControlScroll:
+            return self.mapView.scrollEnabled;
+        case MBXEmbeddedControlRotation:
+            return self.mapView.rotateEnabled;
+        case MBXEmbeddedControlPitch:
+            return self.mapView.pitchEnabled;
+    }
+}
+
+#pragma mark UIScrollViewDelegate methods
+
+- (UIView *)viewForZoomingInScrollView:(UIScrollView *)scrollView
+{
+    return self.mapView;
+}
+
+#pragma mark Class method
+
+
++ (NSString *)titleForControl:(MBXEmbeddedControl) control {
```

Comment:
Need consistent bracket style, which is a new line as per rest of SDK and other parts of this file.